### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by matrix880**

### CODE REVIEW

File: src/main/java/core/basesyntax/Machine.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    String name;
+    int year;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Machine.java]: why no access modifiers?

File: src/main/java/core/basesyntax/Excavator.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    String name;
+    int year;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Excavator.java]: why do you recreate fields that already exist at the parent?

Comment (matrix880) [reply src/main/java/core/basesyntax/Excavator.java]: didnt guess.done.

File: src/main/java/core/basesyntax/Machine.java (hunk 1 above)
Comment (matrix880) [reply src/main/java/core/basesyntax/Machine.java]: done.

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public Bulldozer(String name,int year,String colour) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Bulldozer.java:5]:
```suggestion
    public Bulldozer(String name, int year, String colour) {
```
where do you use color?

File: src/main/java/core/basesyntax/Excavator.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public Excavator(String name, int year, String colour) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Excavator.java:5]: same

File: src/main/java/core/basesyntax/Track.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Track extends Machine {
+
+    public Track(String name, int year, String colour) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Track.java:5]: same

File: src/main/java/core/basesyntax/Bulldozer.java (hunk 3 above)
Comment (matrix880) [reply src/main/java/core/basesyntax/Bulldozer.java:5]: do not understand .i add colour characteristic to bUldozer,track and excavator

Comment (matrix880) [reply src/main/java/core/basesyntax/Bulldozer.java]: done

File: src/main/java/core/basesyntax/Excavator.java (hunk 4 above)
Comment (matrix880) [reply src/main/java/core/basesyntax/Excavator.java]: done

File: src/main/java/core/basesyntax/Track.java (hunk 5 above)
Comment (matrix880) [reply src/main/java/core/basesyntax/Track.java]: done

**Pull request closed without merging**